Source/DafnyStandardLibraries/examples/ConcurrentExamples.dfy

```diff
@@ -0,0 +1,177 @@
+module ConcurrentExamples {
+
+  import opened DafnyStdLibs.Concurrent
+  import opened DafnyStdLibs.Wrappers
+  import opened Helpers
+
+  const p1: nat -> bool := _ => true
+  const p2: (nat, nat) -> bool := (_, _) => true
```

Comment:
If it doesn't make verification a lot harder, we should make these not completely degenerate, to illustrate a bit more why it's useful to have an invariant.